python/pyspark/ml/regression.py

```diff
@@ -619,6 +627,20 @@ def getSubsamplingRate(self):
         """
         return self.getOrDefault(self.subsamplingRate)
 
+    @since("1.4.0")
+    def setFeatureSubsetStrategy(self, value):
```

Comment:
this method should be in the GBT/RandomForest estimator classes (classification and regression) and the old method in RandomForestParams should be deprecated, would you mind doing this?

Reply:
@BryanCutler Thanks for your review. I will modify the code. One question: Shall I only put the ```setFeatureSubsetStrategy``` in GBT/RandomForest? or put both the setter and getter? I looked the ml python code, it seems to me that the getter and setter are always in pairs.

Reply:
`setFeatureSubsetStrategy` should only be in the GBT/RF estimators, while `getFeatureSubsetStrategy` can be in `TreeEnsembleParams` so it is inherited by both the estimators and models.  It's because we don't want methods to set training params in the Model classes.

Reply:
Got it. Thanks a lot!